Manual/index.html

```diff
@@ -104,7 +104,7 @@ <h4><a name="songinst">Song Installation</a></h4>
 Songs/StepMania 5/Springtime/Kommisar - Springtime.mp3
 Songs/StepMania 5/Springtime/spring.png
 Songs/StepMania 5/Springtime/springbn.png
-Songs/StepMania 5/Springtime/Springtime.ssc
+Songs/StepMania 5/Springtime/Springtime.sm

```

Comment:
https://github.com/stepmania/stepmania/blob/master/Songs/StepMania%205/Springtime/Springtime.ssc

ssc is the latest and greatest format and we don't even ship this song with a .sm.